script/Arcueid/Day 8/08_00_ARC08_8.txt

```diff
@@ -603,27 +661,32 @@ She's dressed like a Catholic priest from overseas.
 [sha:9f9008852c8ca42c782de222bdac292fc97bdaa9]{
 -- Page 519, Offset 21073.
 -- 右手に<携|たずさ>えた細い剣と、
-She's carrying a thin sword in her right hand,
+// CG here only shows sword, but she throws like 3 of them 2 pages down?
+In her right hand she carries a thin sword,
 }
 [sha:03df30b1a737a0b9d67588a6cf522e52a232116c]{
 -- Page 519, Offset 21074.
 -- 左に握った、あまりにも場違いすぎる巨大な槍。
-and grasping an enormous spear in her left that stands out like a sore thumb.
+and with her left grasps an enormous spear that sticks out like a sore thumb.
 }
 [sha:431bd7bc98745139b647d6dce27b8ee2430d8e43]{
 -- Page 520, Offset 21075.
 -- ……あのカタチは知っている。
-...I know that stance.
+// Talking about shape of weapon, not shape of Noel
+...I know that shape.
 }
 [sha:e8fc0199a382cc0fa1878e01bd709197a4048adc]{
 -- Page 520, Offset 21076.
 -- いや、以前視界に入った事を、脳の一部が覚えている。
-No, a part of my brain remembers a scene like this from before.
+No, a part of me remembers seeing something like it before.
 }
 [sha:cd2ac1c2b88c3cd215da94dfb96c0700f0fe17cf]{
 -- Page 520, Offset 21077.
 -- ヴローヴとの戦いのおり、あの剣が遠野志貴を氷塊から救った事を、意識していなくとも事実として認識している。
-Back when I fought Vlov, I recognize for a fact that a blade like that saved me from those ice blocks, even if I wasn't aware of it at the time.
+// Don't remember when this happened, might need to check back and make sure the scope of the ice is consistent
+Back during the fight with Vlov,
+ that sword was the one that saved me from those blocks of ice...
```

Comment:
I corrected this in Manolo's PR too: "blade" is right. It's referring to those funny little guys Ciel has.